src/js/components/FieldGroup/FieldGroup.stories.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { select } from "@storybook/addon-knobs/react";
+import FieldGroup from "./FieldGroup";
+import FieldLabel from "../FieldLabel";
+import Input from "../Input";
+import Select from "../Select";
+import Checkbox from "../Checkbox";
+import Icon from "../Icon";
+
+const stories = storiesOf("FieldGroup", module);
+
+const statusOptions = {
+  success: "success",
+  error: "error",
+  default: null
+};
+
+const iconOptions = {
+  left: "left",
+  right: "right",
+  default: "left"
+};
+
+stories.add("FieldGroup with Input text child", () => {
+  const status = select("status", statusOptions);
+
+  return (
+    <div style={{ padding: 20 + "px" }}>
+      <FieldGroup name="example" label="Check Me" isInline status={ status }>
+        <FieldLabel name="example">Text Input</FieldLabel>
+        <Input name="storybook" type="text" />
+      </FieldGroup>
+    </div>
+  );
+});
+
+stories.add("FieldGroup with Select child", () => {
+  const status = select("status", statusOptions);
+
+  return (
+    <div style={{ padding: 20 + "px" }}>
```

Comment:
Hey Kyle, I think this would be better done globally so it's not a part of the story code preview. Definitely wouldn't want this copy and pasted into production. Here's a couple links that look promising for global storybook style additions if you'd like to tackle. I'm fine with just removing the padding and following up with this later too

- https://storybook.js.org/docs/configurations/default-config/#css-support
- https://jaketrent.com/post/global-styles-in-storybook/

Reply:
@l3aconator  took a stab at this and this this seems to work pretty well...

<img width="706" alt="Screen Shot 2019-05-16 at 1 27 53 PM" src="https://user-images.githubusercontent.com/1189536/57881468-98b5b100-77de-11e9-8f14-162faa570eaf.png">
<img width="1077" alt="Screen Shot 2019-05-16 at 1 28 15 PM" src="https://user-images.githubusercontent.com/1189536/57881469-98b5b100-77de-11e9-8d66-b32e7c386ef5.png">